docs/project_description.md

```diff
@@ -0,0 +1,87 @@
+# _GoldenProfile_
```

Comment:
Good point. When I named the German version of the project description `project_description_de.md`, I thought about naming the English description similar.  But since we want to do the project completely in English, the "main language" of the documents should also be English.

I don't think we will have a German version for every document we create in the course of this project. For this reason, I think it would be better to have the main documents without a language suffix and any further version of the document with language suffix.

For example:
- architecture.md (English)
- project_description.md (English)
- project_description_de_md (German)
- structure.md (English)
- workflow.md (English)

Otherwise we would have this structure:
- architecture_en.md (English)
- project_description_en.md (English)
- project_description_de_md (German)
- structure_en.md (English)
- workflow_en.md (English)

There is always the case that documents are not updated correctly. Especially if you have the document in several languages.Therefore it's better to have a main document, which is always up-to-date (yes of course 😄 both documents should always be up-to-date. but how often is this not the case? Very often...)